runtime-sdk/src/event.rs

```diff
@@ -2,6 +2,24 @@
 use oasis_core_runtime::{common::cbor, transaction::tags::Tag};
 
 /// An event emitted by the runtime.
+///
+/// This trait can be derived:
+/// ```ignore
+/// # #[cfg(feature = "oasis-runtime-sdk-macros")]
+/// # mod example {
+/// # use serde::{Serialize, Deserialize};
+/// #[derive(Clone, Debug, Serialize, Deserialize, oasis_runtime_sdk::Event)]
+/// #[event(module = "path::to::MyModule", autonumber)] // `module` is required
```

Comment:
This example has the `event` attribute, actual code has `sdk` :)

Reply:
would you mind me adding [mockall](https://crates.io/crates/mockall) so that I can easily create a mock `Module` and un-`ignore` the test?

Reply:
Yes, that would be great so we could test these examples :-)